src/harness/util.py

```diff
@@ -408,3 +409,69 @@ def filterChannelsByFrequencyRange(channels, freq_range):
       and
       channel['frequencyRange']['highFrequency'] <= freq_range['highFrequency']
   ]
+
+def _orderAttributes(obj):
+    if isinstance(obj, dict):
+        return sorted((k, _orderAttributes(v)) for k, v in obj.items())
+    if isinstance(obj, list):
+        return sorted(_orderAttributes(x) for x in obj)
+    else:
+        return obj
+
+def compareDictWithUnorderedLists(first_dict, second_dict):
+  """ Deep comparison of two dictionaries
+
+  Args:
+    first_dict: first dictionary to be compared.
+    second_dict: second dictionary to be compared.
+ Returns: boolean set to true if the dictionaries are equal and false otherwise.
+  """
+  return _orderAttributes(first_dict) == _orderAttributes(second_dict)
+
+def isPolygonCCW(polygon):
```

Comment:
See my comments at the end of file. If this routine is really needed to check that geojson follows the recommended winding, then  exterior should be CCW and holes should be CW.
Here you are checking that both are CCW.

The function should then be renamed something like:
```
 def  hasPolygonCorrectGeoJsonWinding(polygon).
```

Finally it would be better to move this function in the reference_models.geo.utils module, along with other similar function manipulating geometries, including the unit test.

Reply:
thank you for your good suggestions, I updated the code.